homeassistant/components/lyric/services.yaml

```diff
@@ -1,9 +1,18 @@
 set_hold_time:
+  name: Set Hold Time
   description: "Sets the time to hold until"
+  target:
+    device:
+      integration: lyric
+    entity:
+      integration: lyric
+      domain: climate
   fields:
-    entity_id:
```

Comment:
I think this is an breaking change, because there fields removed and new one added, so existing automatons become broken

Reply:
Yeah you're right. I'll add it to the description

Reply:
Please double check me, but I think `target` should be backwards compatible in most cases since the service calls handler looks for `entity_id`, `area_id`, and `device_id`.

Reply:
Yeah, you are right ... this smart `Service target` approach is quiet new to me (_I did not know it before_)

Reply:
Yeah the target gets set as soon as you update to 2021.3.0 and change your automation. I'll remove the breaking change 😺